packages/documentation/src/stories/foundation/layout/Columns/columns.stories.ts

```diff
@@ -0,0 +1,252 @@
+      </div>
+    </div>
+  `,
+};
+
+export const OrderExample: Story = {
+  render: () => html`
+    <div class="container">
+      <div class="row">
+        <div class="col">First in DOM, no order applied</div>
+        <div class="col order-5">Second in DOM, with a larger order</div>
+        <div class="col order-1">Third in DOM, with an order of 1</div>
+      </div>
+    </div>
+  `,
+};
+
+export const OrderMaxExample: Story = {
+  render: () => html`
+    <div class="container">
+      <div class="row">
+        <div class="col order-last">First in DOM, ordered last</div>
+        <div class="col">Second in DOM, unordered</div>
+        <div class="col order-first">Third in DOM, ordered first</div>
+      </div>
+    </div>
+  `,
+};
+
+export const OffsetExample: Story = {
+  parameters: {
+    controls: { exclude: ['Align Items', 'Align Item 1', 'Horizontal Alignement'] },
+  },
+  render: (args: Args) => html`
+    <div class="row">
+      <div class="col-1 ${args.offsetItem}">.col-1 .${args.offsetItem}</div>
+    </div>
+  `,
+};
+
+export const ColumnBreakExample: Story = {
+  render: () => html`
+    <div class="row">
+      <div class="col-3">.col-3</div>
+      <div class="col-3">.col-3</div>
+
+      <!-- Force next columns to break to new line -->
+      <div class="w-100"></div>
+
+      <div class="col-3">.col-3</div>
+      <div class="col-3">.col-3</div>
+    </div>
+  `,
+};
+
+export const ResetOffsetExample: Story = {
+  decorators: [
+    (story: StoryFn, { args, context }: StoryContext) => html`
+      ${story(args, context)}
+      <p class="mt-regular"><small>Resize the browser window to see changes.</small></p>
+    `,
+  ],
+  render: () => html`
+    <div class="row">
+      <div class="col-sm-5 col-md-6">.col-sm-5 .col-md-6</div>
+      <div class="col-sm-5 offset-sm-2 col-md-6 offset-md-0">
+        .col-sm-5 .offset-sm-2 .col-md-6 .offset-md-0
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-sm-6 col-md-5 col-lg-6">.col-sm-6 .col-md-5 .col-lg-6</div>
+      <div class="col-sm-6 col-md-5 offset-md-2 col-lg-6 offset-lg-0">
+        .col-sm-6 .col-md-5 .offset-md-2 .col-lg-6 .offset-lg-0
+      </div>
+    </div>
+  `,
+};
+
+export const MarginUtilitiesExample: Story = {
+  decorators: [
+    (story: StoryFn, { args, context }: StoryContext) => html`
+      ${story(args, context)}
+      <p class="mt-regular"><small>Resize the browser window to see changes.</small></p>
+    `,
+  ],
+  render: () => html`
+    <div class="row">
+      <div class="col-md-4">.col-md-4</div>
+      <div class="col-md-4 ms-auto">.col-md-4 .ms-auto</div>
+    </div>
+    <div class="row">
+      <div class="col-md-3 ms-md-auto">.col-md-3 .ms-md-auto</div>
+      <div class="col-md-3 ms-md-auto">.col-md-3 .ms-md-auto</div>
+    </div>
+    <div class="row">
+      <div class="col-auto me-auto">.col-auto .me-auto</div>
+      <div class="col-auto">.col-auto</div>
+    </div>
+  `,
+};
```

Comment:
I would recommend to remove this example. Because yes, it's possible to do such things with the margin-utilities, but I would not tell any one to do it this way. Instead DEVs should use the justify, alignment and/or offset classes.